doc/source/dev/roadmap-detailed.rst

```diff
@@ -413,12 +413,12 @@ may also be included in SciPy, especially if no other widely used and
 well-supported package covers the topic.  Also note that *some* duplication
 with downstream projects is inevitable and not necessarily a bad thing.)
 
-The following improvements will help SciPy better serve this role.
+In addition to the items described in the :ref:`scipy-roadmap`, the following
+improvements will help SciPy better serve this role.
 
 - Add fundamental and widely used hypothesis tests:
 
-  - Tukey-Kramer test
-  - Dunnett's test
+  - Dunnett's test and the Student–Newman–Keuls test
   - the various types of analysis of variance (ANOVA):
```

Comment:
I would rephrase this section in favour of something more general on sensitivity analysis.

Reply:
Sure. Could you suggest a specific phrasing?

Reply:
ANOVA and sensitivity analysis have some similarities, but they are distinct things. One can _use_ ANOVA to do sensitivity analysis on mathematical models, but as @tupui suggests, there are other approaches that work better for that purpose. But the functionalities mentioned here are distinct from that use case.

Reply:
@rkern what would you propose?

From what I know/understand of ANOVA, classical sensitivity analysis seems to be a bit more general/insightful on a model/problem. This is why I proposed to broaden the statement and talk about sensitivity analysis instead. Linked to our discussion which IMHO did not really found a conclusion.

Reply:
I would propose leaving the roadmap alone on this point.

ANOVA and sensitivity analysis are just two different things. They aren't alternatives to each other nor is one a specific instance of the other. If you want to add sensitivity analysis functionality, that's an entirely separate topic; you aren't addressing the needs that put ANOVA on this list.

Now, I don't particularly care for ANOVA methods (I'd go straight to Bayesian modeling), and I don't think that scipy is particularly suited to have a zoo of them. I think I would want at least pandas `DataFrame`s and a model description syntax like [`patsy`](https://patsy.readthedocs.io/en/latest/) to provide some of the infrastructure (and indeed, this is what `statsmodels` does). We had an ANOVA implementation a long time ago, but it was very janky and didn't fit in with anything else in `scipy.stats` so we removed it. I think ANOVA in scipy, given those constraints, is largely a waste of everyone's time and energy, but it's not my time and energy that was being volunteered when this was added to the roadmap, so I don't have a veto.

Reply:
My opinion hasn't changed on the subject, no.

Reply:
[This proposal](https://mail.python.org/archives/list/scipy-dev@python.org/message/DBAHHBWLVJQ2X5U2SXFOUIGBSLB4I7RE/) of adding Sobol's indices sounds good to me (since the papers cited meet SciPy's requirements and also isn't too much code to review and add). Maybe, we can add only that part to the roadmap? @stefanv is also [+1](https://mail.python.org/archives/list/scipy-dev@python.org/message/VXLAGHE2VS4JSHFR4ZV44JOX2CLQBSXU/) on the mailing list.

Reply:
Sounds like there is not yet consensus on this, so leave it to another PR.